backend/src/main/scala/cromwell/backend/standard/GroupMetricsActor.scala

```diff
@@ -1,26 +1,39 @@
 package cromwell.backend.standard
 
-import akka.actor.{Actor, ActorLogging, Props}
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
 import akka.dispatch.MessageDispatcher
 import common.util.StringUtil.EnhancedToStringable
-import cromwell.backend.standard.GroupMetricsActor.RecordGroupQuotaExhaustion
+import cromwell.backend.standard.GroupMetricsActor._
 import cromwell.core.Dispatcher
 import cromwell.core.Dispatcher.EngineDispatcher
 import cromwell.database.sql.EngineSqlDatabase
 import cromwell.database.sql.SqlConverters.OffsetDateTimeToSystemTimestamp
 import cromwell.database.sql.tables.GroupMetricsEntry
 
 import java.time.OffsetDateTime
+import scala.util.{Failure, Success}
 
 class GroupMetricsActor(engineDbInterface: EngineSqlDatabase) extends Actor with ActorLogging {
 
   implicit val ec: MessageDispatcher = context.system.dispatchers.lookup(Dispatcher.EngineDispatcher)
 
+  final private val QUOTA_EXHAUSTION_THRESHOLD_IN_SECS = 15 * 60 // 15 minutes
```

Comment:
Question for reviewers - what should be this threshold? Is 15 mins good?

Reply:
How hard would it be to make this configurable? I think 15m is a good place to start, if anything a little long.

Reply:
I don't think it would be that hard. I can try that.

Reply:
Configurable sounds good, yeah.

How long is our polling interval for cloud jobs? It seems like the quota backoff time should be more than the poll interval.

Reply:
>How long is our polling interval for cloud jobs?

It seems the it initially starts with [30 seconds](https://github.com/broadinstitute/cromwell/blob/67ca8aff6a4ec4f3c96e2720f10ab97217718a1a/supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala#L138-L142) and the [max interval is 10 mins](https://github.com/broadinstitute/terra-helmfile/blob/a1026e48af86063f0f65c3d7def5af42587ca09e/charts/cromwell/templates/config/_cromwell-papi-v2-beta-config.conf.tpl#L53). So maybe 15 mins is a good starting point?

Reply:
Agreed, sounds great to me.